diff --git a/mysql-test/suite/perfschema/r/information_schema.result b/mysql-test/suite/perfschema/r/information_schema.result
index 3f4a8cf6336..c0aff9bb956 100644
--- a/mysql-test/suite/perfschema/r/information_schema.result
+++ b/mysql-test/suite/perfschema/r/information_schema.result
@@ -59,6 +59,7 @@ performance_schema	prepared_statements_instances	def
 performance_schema	replication_applier_configuration	def
 performance_schema	replication_applier_status	def
 performance_schema	replication_applier_status_by_coordinator	def
+performance_schema	replication_applier_status_by_worker	def
 performance_schema	replication_connection_configuration	def
 performance_schema	rwlock_instances	def
 performance_schema	session_account_connect_attrs	def
@@ -144,6 +145,7 @@ prepared_statements_instances	BASE TABLE	PERFORMANCE_SCHEMA
 replication_applier_configuration	BASE TABLE	PERFORMANCE_SCHEMA
 replication_applier_status	BASE TABLE	PERFORMANCE_SCHEMA
 replication_applier_status_by_coordinator	BASE TABLE	PERFORMANCE_SCHEMA
+replication_applier_status_by_worker	BASE TABLE	PERFORMANCE_SCHEMA
 replication_connection_configuration	BASE TABLE	PERFORMANCE_SCHEMA
 rwlock_instances	BASE TABLE	PERFORMANCE_SCHEMA
 session_account_connect_attrs	BASE TABLE	PERFORMANCE_SCHEMA
@@ -229,6 +231,7 @@ prepared_statements_instances	10	Dynamic
 replication_applier_configuration	10	Fixed
 replication_applier_status	10	Fixed
 replication_applier_status_by_coordinator	10	Dynamic
+replication_applier_status_by_worker	10	Dynamic
 replication_connection_configuration	10	Dynamic
 rwlock_instances	10	Dynamic
 session_account_connect_attrs	10	Dynamic
@@ -314,6 +317,7 @@ prepared_statements_instances	0
 replication_applier_configuration	0
 replication_applier_status	0
 replication_applier_status_by_coordinator	0
+replication_applier_status_by_worker	0
 replication_connection_configuration	0
 rwlock_instances	0
 session_account_connect_attrs	0
@@ -406,6 +410,7 @@ prepared_statements_instances	0	0
 replication_applier_configuration	0	0
 replication_applier_status	0	0
 replication_applier_status_by_coordinator	0	0
+replication_applier_status_by_worker	0	0
 replication_connection_configuration	0	0
 rwlock_instances	0	0
 session_account_connect_attrs	0	0
@@ -491,6 +496,7 @@ prepared_statements_instances	0	0	NULL
 replication_applier_configuration	0	0	NULL
 replication_applier_status	0	0	NULL
 replication_applier_status_by_coordinator	0	0	NULL
+replication_applier_status_by_worker	0	0	NULL
 replication_connection_configuration	0	0	NULL
 rwlock_instances	0	0	NULL
 session_account_connect_attrs	0	0	NULL
@@ -576,6 +582,7 @@ prepared_statements_instances	NULL	NULL	NULL
 replication_applier_configuration	NULL	NULL	NULL
 replication_applier_status	NULL	NULL	NULL
 replication_applier_status_by_coordinator	NULL	NULL	NULL
+replication_applier_status_by_worker	NULL	NULL	NULL
 replication_connection_configuration	NULL	NULL	NULL
 rwlock_instances	NULL	NULL	NULL
 session_account_connect_attrs	NULL	NULL	NULL
@@ -661,6 +668,7 @@ prepared_statements_instances	utf8_general_ci	NULL
 replication_applier_configuration	utf8_general_ci	NULL
 replication_applier_status	utf8_general_ci	NULL
 replication_applier_status_by_coordinator	utf8_general_ci	NULL
+replication_applier_status_by_worker	utf8_general_ci	NULL
 replication_connection_configuration	utf8_general_ci	NULL
 rwlock_instances	utf8_general_ci	NULL
 session_account_connect_attrs	utf8_bin	NULL
@@ -746,6 +754,7 @@ prepared_statements_instances
 replication_applier_configuration	
 replication_applier_status	
 replication_applier_status_by_coordinator	
+replication_applier_status_by_worker	
 replication_connection_configuration	
 rwlock_instances	
 session_account_connect_attrs	
@@ -831,6 +840,7 @@ prepared_statements_instances
 replication_applier_configuration	
 replication_applier_status	
 replication_applier_status_by_coordinator	
+replication_applier_status_by_worker	
 replication_connection_configuration	
 rwlock_instances	
 session_account_connect_attrs
