diff --exclude .svn -ur mecab.orig/configure.in mecab/configure.in
--- mecab.orig/configure.in	2012-09-20 13:46:17.431392052 +0900
+++ mecab/configure.in	2012-09-20 15:09:21.695405072 +0900
@@ -47,6 +47,7 @@
 AC_CHECK_SIZEOF(long long)
 AC_CHECK_SIZEOF(size_t)
 AC_TYPE_SIZE_T
+AC_TYPE_UNSIGNED_LONG_LONG_INT
 
 AC_FUNC_MMAP
 AC_CHECK_LIB(stdc++, main, MECAB_LIBS="-lstdc++")
diff --exclude .svn -ur mecab.orig/src/string_buffer.h mecab/src/string_buffer.h
--- mecab.orig/src/string_buffer.h	2012-09-20 13:46:15.875392048 +0900
+++ mecab/src/string_buffer.h	2012-09-20 15:09:46.783405140 +0900
@@ -44,6 +44,9 @@
   StringBuffer& operator<<(unsigned short int n) { _UITOA(n); }
   StringBuffer& operator<<(unsigned int n)       { _UITOA(n); }
   StringBuffer& operator<<(unsigned long int n)  { _UITOA(n); }
+#ifdef HAVE_UNSIGNED_LONG_LONG_INT
+  StringBuffer& operator<<(unsigned long long int n) { _UITOA(n); }
+#endif
 
   StringBuffer& operator<< (char n) {
     return this->write(n);
