--- mecab-0.994.orig/src/thread.h	2011-12-05 02:31:09.000000000 +0900
+++ mecab-0.994/src/thread.h	2012-09-20 12:15:46.695377866 +0900
@@ -47,7 +47,7 @@
 #undef yield_processor
 #define atomic_add(a, b) ::InterlockedExchangeAdd(a, b)
 #define compare_and_swap(a, b, c)  ::InterlockedCompareExchange(a, c, b)
-#define yield_processor() ::YieldProcessor()
+#define yield_processor() YieldProcessor()
 #define HAVE_ATOMIC_OPS 1
 #endif
 
