diff -ru mecab-0.98.orig/src/libmecab.cpp mecab-0.98/src/libmecab.cpp
--- mecab-0.98.orig/src/libmecab.cpp	2009-04-19 00:03:04.000000000 +0900
+++ mecab-0.98/src/libmecab.cpp	2011-04-29 23:45:03.331006297 +0900
@@ -57,9 +57,11 @@
     if (!DllInstance) {
       DllInstance = hinst;
     }
+#if !defined(__GNUC__)
     std::locale loc(std::locale("japanese"),
                     "C", std::locale::numeric);
     std::locale::global(loc);
+#endif
     return TRUE;
   }
 #ifdef __cplusplus
