diff -ru mecab-0.98.orig/src/Makefile.am mecab-0.98/src/Makefile.am
--- mecab-0.98.orig/src/Makefile.am	2008-09-14 02:01:17.000000000 +0900
+++ mecab-0.98/src/Makefile.am	2014-08-28 22:16:44.262074429 +0900
@@ -3,6 +3,7 @@
 EXTRA_DIST = Makefile.msvc.in make.bat
 pkglibexecdir = ${libexecdir}/mecab
 INCLUDES = -DDIC_VERSION=$(DIC_VERSION) $(MECAB_WITHOUT_SHARE_DIC) $(MECAB_WITHOUT_MUTEX_LOCK) $(MECAB_USE_UTF8_ONLY) -DMECAB_DEFAULT_RC="\"$(MECAB_DEFAULT_RC)\""
+libmecab_la_CPPFLAGS = -DDLL_EXPORT
 libmecab_la_LDFLAGS = -no-undefined -version-info $(LTVERSION)
 libmecab_la_SOURCES = utils.cpp utils.h eval.cpp iconv_utils.cpp iconv_utils.h \
                       dictionary_rewriter.h dictionary_rewriter.cpp dictionary_generator.cpp \
