diff -ru mecab-0.98.orig/src/string_buffer.h mecab-0.98/src/string_buffer.h
--- mecab-0.98.orig/src/string_buffer.h	2009-04-19 00:03:04.000000000 +0900
+++ mecab-0.98/src/string_buffer.h	2011-04-24 09:40:23.166985912 +0900
@@ -45,6 +45,9 @@
   StringBuffer& operator<<(unsigned short int n) { _UITOA(n); }
   StringBuffer& operator<<(unsigned int n)       { _UITOA(n); }
   StringBuffer& operator<<(unsigned long int n)  { _UITOA(n); }
+#if SIZEOF_SIZE_T == SIZEOF_LONG_LONG
+  StringBuffer& operator<<(unsigned long long int n) { _UITOA(n); }
+#endif
 
   StringBuffer& operator<< (char n) {
     return this->write(n);
