diff -ur mecab-0.994.orig/configure.in mecab-0.994/configure.in
--- mecab-0.994.orig/configure.in	2012-04-01 23:21:02.000000000 +0900
+++ mecab-0.994/configure.in	2012-09-14 15:58:19.759200168 +0900
@@ -302,6 +302,8 @@
 case "$host_os" in
  mingw* | os2*)
   MECAB_DEFAULT_RC='c:\\\\Program Files\\\\mecab\\\\etc\\\\mecabrc'
+  CFLAGS="$CFLAGS -municode -DUNICODE -D_UNICODE"
+  CXXFLAGS="$CXXFLAGS -municode -DUNICODE -D_UNICODE"
   ;;
  *)
   MECAB_DEFAULT_RC="$sysconfdir/mecabrc"
